records/scrum_team_charter.md

```diff
@@ -0,0 +1,87 @@
+# Visual Meaning Scrum Team Charter
+
+## Things we always do
+
+- Try to write the best code
+
+- **Be kind**
+  - Assume best intent
+  - Bring humanity to work - full spectrum of emotion and acknowledgement that we are humans who do have personal lives
+  - Reach out and be vulnerable
+
+- **Commit**
+  - Do what we say we will do (and communicate when things change)
+  - Keep practical details front of mind
+  - Prioritise team outcomes
+  - Be meaningfully wrong
+
+- **Collaborate**
+  - Teach, share information and decisions, learning early and often and recording this where people can find the information again
+  - Broadcast by default
+
+- **Engage**
+  - Engage in synchronous sessions

```

Comment:
I have some issues with how team and the company are using the synchronous sessions. Also I have strong reactions when people say that I'm not engaged. So it looks to me that it could lead to misinterpretations. I already read one from AL about this and prefer to not have a subjective way of accusing me form something. I will not sign this and I will not follow this bullet point in this shape.